server/templates/home.html

```diff
@@ -3,10 +3,6 @@
 
 {% block title %}Home{% endblock %}
 
-{% if navOptions != None %}
-{% block nav %}{% endblock %}
-{% endif %}
-
```

Comment:
I'm confused. Did we just rip the nav bar out entirely?

Reply:
No, the navbar is used in layout.html, and the navbar code is its own template rather than a Jinja block. So this code was obsolete.